src/OpenColorIO/ParseUtils.cpp

```diff
@@ -265,7 +265,7 @@ const char * GpuLanguageToString(GpuLanguage language)
         case GPU_LANGUAGE_GLSL_4_0:     return "glsl_4.0";
         case GPU_LANGUAGE_GLSL_ES_1_0:  return "glsl_es_1.0";
         case GPU_LANGUAGE_GLSL_ES_3_0:  return "glsl_es_3.0";
-        case GPU_LANGUAGE_HLSL_DX11:    return "hlsl_dx11";
+        case GPU_LANGUAGE_HLSL_SM_5_0:  return "hlsl_sm_5.0";
```

Comment:
Don't remember... so let me know if any precautions are needed here as this change would cause cache id invalidation.